pkg/cli/cmds/server.go

```diff
@@ -10,77 +10,77 @@ var (
 	config rke2.Config
 
 	k3sServerBase = mustCmdFromK3S(cmds.NewServerCommand(ServerRun), map[string]*K3SFlagOption{
-		"v":                 Hide,
-		"vmodule":           Hide,
-		"log":               Hide,
-		"alsologtostderr":   Hide,
+		"v":                 hide,
+		"vmodule":           hide,
+		"log":               hide,
+		"alsologtostderr":   hide,
 		"bind-address":      nil,
-		"https-listen-port": Drop,
+		"https-listen-port": drop,
 		"advertise-address": nil,
-		"advertise-port":    Drop,
+		"advertise-port":    drop,
 		"tls-san":           nil,
 		"data-dir": {
```

Comment:
I wonder if we can set this to `nil`.  This commit https://github.com/rancher/k3s/commit/7e59c0801e905f9f574264326ba42143c665b3b5 was supposed to make all instances of the string "k3s" be changed to "rke2" at compile time. So either setting this flag is not needed or the referenced commit does not do all it should.  So ideally this should be `nil`.